website/common/script/content/constants/events.js

```diff
@@ -72,24 +72,39 @@ export const REPEATING_EVENTS = {
     ],
   },
   namingDay: {
-    start: new Date('1970-07-30T08:00-05:00'),
-    end: new Date('1970-08-01T23:59-05:00'),
+    start: new Date('1970-07-30T04:00-05:00'),
+    end: new Date('1970-08-01T03:59-05:00'),
     foodSeason: 'Cake',
   },
+  fallGemFest: {
+    start: new Date('1970-09-23T04:00-04:00'),
+    end: new Date('1970-09-27T03:59-04:00'),
+    gemsPromo,
+  },
+  spookyGemFest: {
+    start: new Date('1970-10-28T04:00-04:00'),
+    end: new Date('1970-10-27T03:59-04:00'),
+    gemsPromo,
+  },
   habitoween: {
-    start: new Date('1970-10-30T08:00-05:00'),
-    end: new Date('1970-11-01T23:59-05:00'),
+    start: new Date('1970-10-30T04:00-05:00'),
+    end: new Date('1970-11-01T23:059-05:00'),
```

Comment:
The extra 0 here messes up the date parsing causing the end date to run into October